components/ILIAS/Administration/classes/Setup/class.ilAdministrationDBUpdateSteps.php

```diff
@@ -75,4 +75,42 @@ public function step_4(): void
     {
         $this->db->manipulate("DELETE FROM settings WHERE keyword LIKE 'char_selector%'");
     }
+
+    public function step_5(): void
```

Comment:
Please move this to an own database objective in the `Authentication` component. You can name it `AbandonLoadDependantSessionDatabaseUpdateObjective`.